lib/backend/k8s/k8s.go

```diff
@@ -94,6 +95,9 @@ func NewKubeClient(kc *KubeConfig) (*KubeClient, error) {
 			*override.variable = override.value
 		}
 	}
+	if kc.K8sInsecureSkipTLSVerify {
```

Comment:
For consistency, probably best to put this in the existing overrides mechanism above.

https://github.com/projectcalico/libcalico-go/pull/356/files#diff-cfc4259ee76e5c99f93b10127d9736beR78